content/blog/pulumi-2-0-update/index.md

```diff
@@ -0,0 +1,76 @@
+
+## Go
+
+We've overhauled [our Go SDK ]({{< relref "/docs/intro/languages/go" >}})to be more idiomatic and easier to use. The changes here are a major improvment as you can see [in this before/after PR](https://github.com/pulumi/examples/commit/b88f385a8951f81fe41cb4f40eb940b074e9263b#diff-07803f7bc3bff97871e6e3eea7287cb8). We previously used `map[string]interface{}` types in many places instead of properly-typed data structures. Not any more &mdash; now you get strong typing for all Go resource APIs across all providers, including AWS, Azure, GCP, and Kubernetes. Examples and documentation have been converted to this new style. [This blog post]({{< relref "/blog/go-sdk-road-to-2" >}}) provides an overview of this new programming model approach.
+
+Although we've had GoDoc support for some time now, we never linked to them! [Now we do.]({{< relref "/docs/reference/pkg?language=golang ">}}) We've also added Go versions to our getting started guides for [AWS]({{< relref "/docs/get-started/aws?language=golang" >}}), [Azure]({{< relref "/docs/get-started/azure?language=golang" >}}), and [GCP]({{< relref "/docs/get-started/gcp?language=golang" >}}), and have many more improvements on the way.
+
+There are a few core features still remaining to flesh out. The biggest one is retiring Dep in favor of Go modules everywhere. We are also adding support for aliases, transformations, and improved API projections, such as supporting union types. Finally, we'll be adding Go support for our Kubernetes, Docker, and Terraform remote state packages.
+
+## Policy as Code
+
+[CrossGuard, our Policy as Code framework]({{< relref "/docs/guides/crossguard" >}}), is now ready to take for a serious test drive. This feature enables you to write policies in real code and enforce them during updates. Policies can check for anything, however common checks include security, compliance, cost management, and general and team best practices. You can write your own policy packages or use off-the-shelf policy packs like our own AWSGuard package.
+
+CrossGuard is open source and the functionality is behind [the new `pulumi policy` command]({{< relref "/docs/reference/cli/pulumi_policy" >}}), as well as [the new `--policy-pack` flag]({{< relref "/docs/reference/cli/pulumi_up#options" >}}) for the `preview` and `up` commands. You can use this on any edition of Pulumi, without restriction, including the offline backends.
+
+If you choose to use CrossGuard with the Enterprise Edition of Pulumi, however, you'll get some added functionality. This includes server-side enforcement of policies and organizational policies, including "policy groups" which let you group and apply many policies together to your stacks, such as for instance applying different policies to production than development environments.
```

Comment:
Is it really a "server-side" enforcement? I through policies are still evaluated inside `pulumi up`?

Reply:
The terminology we use is that the server _enforces_ the rules that get run, although the client _evaluates_ them.